eth-contracts/contracts/service/ServiceTypeManager.sol

```diff
@@ -47,25 +53,7 @@ contract ServiceTypeManager is RegistryContract {
         RegistryContract.initialize();
     }
 
-    function setServiceVersion(
```

Comment:
majority of the diff in this contract is just re-ordering of functions to separate type and version logic